### DISCUSSION TITLE
Proper Uninstall of Vanjaro

### DISCUSSION THREAD

**Discussion by dgingold** [vanjarosoftware/Vanjaro.Platform#2315] [General]
I can't find any docs on how to remove Vanjaro from a DNN site. I've located a listing of it in Libraries under extensions and see a delete icon but not sure it's that simple. Will it put my account login module back the way it found it? Or is it better to take my other content and start a fresh instance of DNN?

**Comment by mastercodefu** [comment]
Any details on how to do this?
